infra/app/service/main.tf

```diff
@@ -153,6 +153,8 @@ module "service" {
   extra_environment_variables = merge({
     FEATURE_FLAGS_PROJECT = module.feature_flags.evidently_project_name
     BUCKET_NAME           = local.storage_config.bucket_name
+    ENVIRONMENT           = var.environment_name
+    BUILD_DATE            = timestamp()
```

Comment:
FYI: `BUILD_DATE` will actually reflect the deploy date.

Reply:
I was going by the description in the ticket, do you think we should rename the variable or keep it as is?

Reply:
Keep it based on the ticket's AC:
> The build_date reflects the date the app was last deployed